src/sagemaker/fw_utils.py

```diff
@@ -140,6 +149,19 @@ def _is_tf_14_or_later(framework, framework_version):
     )
 
 
+def _is_pt_12_or_later(framework, framework_version):
+    """
+    Args:
+        framework: Name of the frameowork
+        framework_version: framework version
+    """
+    # Asimov team now owns PyTorch 1.2.0 py2 and py3
+    asimov_lowest_pt = [1, 2, 0]
```

Comment:
I think "asimov" is an internal name, and probably shouldn't be used here

Reply:
+1